src/gui/threejsScene.js

```diff
@@ -89,8 +94,10 @@ import {setMatrixFromArray} from "./scene/utils.js";
         const domElement = document.getElementById('mainThreejsCanvas');
         mainRenderer = new Renderer(domElement);
 
-        mainCamera = new DefaultCamera("mainCamera", window.innerWidth / window.innerHeight);
-        mainRenderer.add(mainCamera); // Normally not needed, but needed in order to add child objects relative to camera
+        defaultCamera = new DefaultCamera("Default Camera", window.innerWidth / window.innerHeight);
+        webXRCamera = null; // can only be initilized if we have a webxr session
+        mainRenderer.add(defaultCamera); // Normally not needed, but needed in order to add child objects relative to camera
```

Comment:
I don;t know if we should also add the internally managed webxr camera for the same reason, but attaching things to the webxr camera might be a bad idea on its own